README.md

```diff
@@ -2,6 +2,7 @@ ObjectMapper
 ============
 [![CocoaPods](https://img.shields.io/cocoapods/v/ObjectMapper.svg)](https://github.com/Hearst-DD/ObjectMapper)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![SwiftPM Compatible](https://img.shields.io/badge/SwiftPM-Compatible-brightgreen.svg)](https://swift.org/package-manager/")
```

Comment:
I'm working on a new icon.  I'll use the full name for now, although I think people are going to use SPM pretty commonly once it's get going.